mysql_fdw.c

```diff
@@ -942,11 +950,15 @@ mysqlGetForeignPlan(PlannerInfo *root,RelOptInfo *baserel, Oid foreigntableid, F
 	 * field of the finished plan node; we can't keep them in private state
 	 * because then they wouldn't be subject to later planner processing.
 	 */
-	return make_foreignscan(tlist,
-							local_exprs,
-							scan_relid,
-							params_list,
-							fdw_private);
+	return make_foreignscan(tlist
```

Comment:
This branch is based on my PG_95 branch which should have fixed the compile time errors that you listed (please see commit 5f8e57cf). What branch are you on, and is 5f8e57cf in the commit log?